### DISCUSSION TITLE
TableRow DragStart doesn't fire?!

### DISCUSSION THREAD

**Discussion by NeoCoderMatrix86** [Megabit/Blazorise#2061] [Q&A]
Hello :),
I have a Table and want to make a draggable row for reordering items. Therefore I need Events like DragStart, Drop, maybe DragOver. But the DragStart event doesn't fire.
Some example code:
```
<Table Bordered="true" Responsive="true" Narrow="true">
  <TableHeader ThemeContrast="ThemeContrast.Light">
    <TableRow>
[...]
    </TableRow>
  </TableHeader>
  <TableBody>
    <TableRow Draggable="true" DragStart="OnTrackTableDragStartHandler">
[...]
    </TableRow>
  </TableBody>
</Table>
```
Method:
```
private void OnTrackTableDragStartHandler(DragEventArgs args)
{
    Logger.LogDebug("OnTrackTableDragStartHandler with {0}", args);
}
```

I'm using Edge and Version 0.9.3 of Blazorise. When I try to move the row, nothing happens (also no breakpoint is hit). Any ideas, how to do a example of a table with row drag n drop?
Thanks for your help :)

**Comment by stsrki** [comment]
Actually, it works. I just tried. But the problem is that cell is in the way. It is interfering with drag events. Try to drag row at the very start of the table row, near the border. The area is just a few pixels wide.

**Reply by NeoCoderMatrix86** [reply]
Well, I'm not that deep into HTML and CSS (one of the main reasons to use Blazor ;)). Is there a possibility to tell all elements inside that the drag event of the row should be used and fired? Basically it's just simple drag row x to row y, so just 2 events should do the whole logic (if I'm not totally dumb :) ).

**Reply by stsrki** [reply]
This seems to work better

```
<TableRow Draggable="true" DragStart="OnTrackTableDragStartHandler" @ondragstart:stopPropagation="true">
  <TableRowCell >TEST</TableRowCell>
</TableRow>
```

**Comment by stsrki** [comment]
Maybe you can use cell events instead?
